.github/dependabot.yml

```
@@ -11,3 +11,9 @@ updates:
       interval: "daily"
     labels:
       - "area: dependencies"
+  - package-ecosystem: github-actions
```

Actually I missed this issue initially, seems like this needs to be a string.
```suggestion
  - package-ecosystem: "github-actions"
```

---

In yaml, everything is a string by default?

---

Ah, I was going off of [these docs](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot) which had the string quoted. Alright then, although for consistency, it should probably still be quoted, just like in the above `package-ecosystem: "pip"`

---

Done [here](https://github.com/py-mine/mcstatus/compare/4e6e7a426d219d2a70e760484794b5c32bdb82c4..8b58ed95745369197d14f330ecd3ee7899b87157).